Broker services and referral update

Mantelijo opened the pull request.

b1674927 reviewed (changes requested):
- please remove commented code
- please report whether the http://127.0.0.1:8000/executor endpoint reveals the address corresponding to the key you submitted

Mantelijo commented:
**Change Description:**
This PR introduces functionality to update broker-server services as well as referral service on swarm.

**Expected Outcomes:**

Services are updated to latest or selected version (swarm only). Private keys are resubmitted by user and updated correctly. Private keys should be encrypted after services start

**Impact Analysis:**

Should not impact existing deployment setup, only the update functionality.

**Test Cases:**

- Manually deploy older versions of services and update them. Inspect the services' image hashes and verify updates.
- Inspect the /executor endpoint to verify the private key address is correct

**Test Results:**  

Updated services' image hashes change

Broker service /broker-address returns correct wallet address after update
Referral /executor returns correct wallet address after update
